ftw/subsite/tests/test_contact.py

```diff
@@ -33,13 +34,13 @@ def test_contact_everything_ok(self, browser):
         self.assertEqual(len(mh.messages), 1)
 
         msg = email.message_from_string(mh.messages[0])
-        self.assertEqual(msg.get('From'), 'blubber@blubb.ch')
+        self.assertEqual(msg.get('From'), 'Subsite <blubber@blubb.ch>')
         self.assertEqual(msg.get('reply-to'), u'hans peter <test@test.com>')
         sub = msg.get('Subject')
-        self.assertEqual(sub.encode('utf8'), 'Testsubject')
+        self.assertEqual(sub.encode('utf8'), '=?utf-8?q?Testsubject?=')
         self.assertEqual(
             ('hans peter (test@test.com) sends you a message from your site '
-             'Subsite(http:=\n//nohost/plone/subsite):\n'
+             'Subsite (http=\n://nohost/plone/subsite):\n'
```

Comment:
This is not directly related to your change, but what does the `\n` do in `(http=\n://nohost/plone/subsite)`?

Will it cause a line break between `http` and the remainder of the URL?

Reply:
Great Idea! I didn't think about it, narrow minded from a day looking at the same problem.

Reply:
I'm not certain. It definitely should be a line break because the mail is sent plain text. The whitespace I introduced would also explain why it moved to the front. But why it moved by two chars I don't get as well.

Reply:
Relating to our discussion, I think this is file like that. I extracted the change to its own commit to have a better overview.